src/zupply.cpp

```diff
@@ -12412,10 +12412,23 @@ namespace zz
 			const std::size_t alignment = 26;
 			std::string ret;
 
+			std::string type;
```

Comment:
You can do `type=fmt::rtrim(type_)` instead of loop: http://zhreshold.github.io/zupply/namespacezz_1_1fmt.html

Reply:
If I understand correctly what the `fmt::rtrim(type_)` does, I gather it removes all trailing whitespace.
But since I use a newline character (the normal `\n`, not those special CRLF find of newlines) to represent where to separate the different `type_` options, I just want to display it properly for the `get_info()` function without printing on too much new lines, which explains my need to replace **all** newlines that occur within this string and not just the trailing ones.

The newline character was chosen, because in my opinion it shouldn't normally be in the `type_` info. Also I wanted to keep this fix quite small and without too much adaptions, though I believe it might also be possible to store the `type_` as a vector of strings and discarding the idea of storing it within the same string, as I'm doing here.